packages/babel-jest/src/__tests__/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "../../../../tsconfig.json",
+  "compilerOptions": {
+    "composite": false,
+    "esModuleInterop": true,
```

Comment:
do we need this? (applies to all)

Reply:
Some tests needs it. For example, commenting out `"esModuleInterop": true` give type error here:

https://github.com/facebook/jest/blob/ae8cf9a78cb0cd2f54d3bacbc1c8837e540e21dc/packages/pretty-format/src/__tests__/react.test.tsx#L8

My idea was to use the same `tsconfig.json` file for all packages to simplify review. But I can go through and enable it case by case. This option applies only for files inside `__tests__` directories. Perhaps it is fine to keep it?

Reply:
A separate PR making it unneeded would be awesome, but shouldn't clutter up _this_ pr